weave/Scripts/Player.cs

```diff
@@ -123,6 +126,36 @@ private void Rotate(double delta)
         }
     }
 
+    private void HandleIndicators()
```

Comment:
I'm thinking if this logic shouldnt be done by the player? But its pretty modular /easy to extract the way you did it so its fine : )

Reply:
Was trying to do a separate Indicator class but it seemed too complicated